src/simplify.rs

```diff
@@ -71,10 +71,9 @@ pub fn simplify(mut input: String, is_chat_message: bool) -> (String, bool, Opti
     let lines = split_lines(&input);
     let (lines, is_forwarded) = skip_forward_header(&lines);
 
+    let (lines, top_quote) = remove_top_quote(lines);
```

Comment:
Code is moved here to make sure `original_lines` do not include the quote. Otherwise quote is displayed twice, in the quote and in the body.